### DISCUSSION TITLE
Bloaty Nosy not working

### DISCUSSION THREAD

**Discussion by crhule** [builtbybel/Bloatynosy#106] [Q&A]
Hi Community,

Looking forward to debloating my window 11. I download the software, extracted and run the program. I get pop up windows asking if I want to run the program and if I trust it. I click on yes. Then nothing happens. What do I need to do on my end so the program runs. I removed and downloaded the software twice and tried setting the control access to allow any program to run. That didn't work. Help...

**Comment by McKenning** [comment]
You need to "unblock" the exe as it is a downloaded file and smart screen disabled access automatically. To do this, right-click the file, then choose Properties and at the bottom of the general tab, check the "unblock" tick box.
